language/diem-framework/modules/NetworkIdentity.move

```diff
@@ -133,23 +163,27 @@ module DiemFramework::NetworkIdentity {
 
         let has_change = remove_members_internal(identities, &to_remove);
         if (has_change) {
-            Event::emit_event(&mut identity.identity_change_events, NetworkIdentityChangeNotification {
-                identities: *&identity.identities,
-                time_rotated_seconds: DiemTimestamp::now_seconds(),
-            });
+            Event::emit_event(
+                &mut borrow_global_mut<NetworkIdentityEventHandle>(@TreasuryCompliance).identity_change_events,
+                NetworkIdentityChangeNotification {
+                    identities: *&identity.identities,
```

Comment:
seems we are missing the signers account /  address?

Reply:
Yeah, this is missing.  I've fixed it.